core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -188,4 +208,35 @@ class BillingProfileManagerDAOImpl(
     } yield billingProjects.filter(bp => samUserResourceIds.contains(bp.projectName.value))
   }
 
+  private def getProfileApiPolicy(samRole: ProjectRole): String =
+    samRole match {
+      case ProjectRoles.Owner => "owner"
+      case ProjectRoles.User  => "user"
+    }
+
+  def addProfilePolicyMember(billingProfileId: UUID,
```

Comment:
It'd be good to get coverage on these methods in the related [spec](https://github.com/broadinstitute/rawls/blob/develop/core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAOSpec.scala)

Reply:
I added coverage for the new methods. The old ones are covered in integration tests (which we can't do yet for Azure).

Reply:
Done.